pkg/operator/rules_test.go

```diff
@@ -227,7 +228,7 @@ func shouldAcceptRuleWithKeepFiringForPrometheus(t *testing.T) {
 	}
 }
 
-func shouldDropKeepFiringForThanos(t *testing.T) {
+func shouldDropRuleFiringForThanos(t *testing.T) {
```

Comment:
Not a big deal but you could have kept the original name?

Reply:
I made this by example of Prometheus - [Accept and Drop](https://github.com/prometheus-operator/prometheus-operator/pull/6283/files/843d1c954b7127814897b6774daeec4066246237#diff-1b88b230701ac430ad9d7d1396568e0e2023aa33739acf2d9c7d604c82af35acR40).
And, I guess, because of now it's possible to use and that option can be Accepted and should be Dropped in case of version of thanos is "old".